addons/pos_stripe/static/src/js/payment_stripe.js

```diff
@@ -195,8 +197,12 @@ let PaymentStripe = PaymentInterface.extend({
 
     captureAfterPayment: async function (processPayment, line) {
         let capturePayment = await this.capturePayment(processPayment.paymentIntent.id);
-        if (capturePayment.charges)
-            line.card_type = capturePayment.charges.data[0].payment_method_details.card_present.brand;
```

Comment:
would adding `?` achieve the same result?
```js
            line.card_type = capturePayment.charges.data[0].payment_method_details.card_present?.brand;
```

Reply:
@vlst-odoo is this notation accepted in PoS ? I found nowhere in PoS (nor PoS modules) where it is used :thinking: